packages/kolibri-common/components/SearchFiltersPanel/AccordionSelectGroup.vue

```diff
@@ -0,0 +1,468 @@
+<template>
+
+  <div>
+    <AccordionContainer
+      v-if="Object.keys(availableLibraryCategories).length"
```

Comment:
Is there a posibility that all these conditions arent met and we have an empty div for `AccordionSelectGroup`? I suspect if we dont have any of these we wouldnt even get to this point, but just curious if we have considered this.